### DISCUSSION TITLE
mypy complains about `CONFIG_KWARGS` in `UvicornWorker` subclass

### DISCUSSION THREAD

**Discussion by ukalwa** [Kludex/uvicorn#1745] [Potential Issue] [answered]
`mypy` seems to infer the type of `CONFIG_KWARGS` ([in UvicornWorker](https://github.com/encode/uvicorn/blob/master/uvicorn/workers.py#L20)) as `dict[str, str]` as shown below. 
<img width="403" alt="image" src="https://user-images.githubusercontent.com/11986128/198896827-aa0ff8e2-b745-4f63-a509-f92321b69e6f.png">

That seems to cause a problem when I want to add `factory` arg to the config in a subclass
<img width="763" alt="image" src="https://user-images.githubusercontent.com/11986128/198896970-1cd21652-1695-4a27-a8a6-c2edf58730c6.png">

I think the fix is to explicitly annotate CONFIG_KWARGS as `dict[str, Any] or dict[str, bool | str]`

**Comment by ukalwa** [comment, marked as answer]
Created a PR to fix it https://github.com/encode/uvicorn/pull/1746

**Reply by Kludex** [reply]
The above was merged. Thank you. :pray:
